### PR TITLE
Initial draft [PLOPS-427]

### PR DESCRIPTION
**Pull request by mikemonkers**

### CODE REVIEW

File: Docker_Image_Build/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          registry: docker.pkg.github.com/your_repository_in_lower_case/
+```
+
+Mandatory argument:
+
+```yaml
+image_name: "<your image name>"
+```
+
+Option input values (and defaults):
```
Comment (dannystaple) [inline Docker_Image_Build/README.md:46]: Optional?

Comment (mikemonkers) [reply Docker_Image_Build/README.md]: Done

File: Docker_Image_Build/action.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+---
+name: 'Docker Build Images'
+description: 'Promote a branch to a release branch'
+
+inputs:
+  dockerfile:  # Name of dockerfile to build
+    description: 'Dockerfile to build'
```
Comment (dannystaple) [inline Docker_Image_Build/action.yaml:7]: Name or path to?

Comment (mikemonkers) [reply Docker_Image_Build/action.yaml:7]: Quite!

File: Docker_Image_Build/action.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    default: 'Dockerfile'
+  image_name: # Name of image to build.
+    description: 'Docker image name (without registry or tag)'
+    required: true
+  tag_name:
+    description: "Docker image tag to pull"
+    required: false
+    default: "latest"
+  registry: # registry containing the container image
+    description: 'docker registry for container image (must me all lover case
```
Comment (dannystaple) [inline Docker_Image_Build/action.yaml:18]: Typos: 

* "lover" instead of "lower"
* "me" instead of "be".

Comment (mikemonkers) [reply Docker_Image_Build/action.yaml:18]: Done

File: Docker_Image_GetTag/get_git_tag.sh [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+GITHUB_REF=${GITHUB_REF:-""}
+GITHUB_HEAD_REF=${GITHUB_HEAD_REF:-""}
+
+REF="$GITHUB_REF"
+if [ "$GITHUB_HEAD_REF" != "" ]; then
+  REF="$GITHUB_HEAD_REF"
+fi
+if [ "$REF" == 'refs/heads/master' ]
+then
```
Comment (dannystaple) [inline Docker_Image_GetTag/get_git_tag.sh:12]: Inconsistent? One then inline, one on another line?

File: Git_Normalise_Tag/README.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
+# GitHub Action for Promoting a branch
+
+GitHub Action implementation for normalising the latest GitHub Tab to
+be of sematic-release form (eg. v1.2.1).
```
Comment (dannystaple) [inline Git_Normalise_Tag/README.md:4]: That should probably be "semantic-version" form.

File: Docker_Image_GetTag/get_git_tag.sh (hunk 4 above)
Comment (mikemonkers) [reply Docker_Image_GetTag/get_git_tag.sh]: Done

File: Git_Normalise_Tag/README.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    if: github.ref == 'refs/heads/devel'
+    name: "Normalise"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: MasterOfMalt/Atom.DevOps.GitHubActions/Git_Normalise_Tag@v1
+        if: env.ACT != 'true'
+```
+
+Option input values (and defaults):
```
Comment (dannystaple) [inline Git_Normalise_Tag/README.md:30]: Missed one optional.

Comment (mikemonkers) [reply Git_Normalise_Tag/README.md:38]: Done

File: Git_Normalise_Tag/README.md (hunk 5 above)
Comment (mikemonkers) [reply Git_Normalise_Tag/README.md]: Done

**Review by dannystaple — APPROVED** [review]

**Pull request merged**